conans/client/tools/env.py

```diff
@@ -46,6 +53,8 @@ def environment_append(env_vars):
     if env_vars:
         old_env = dict(os.environ)
         os.environ.update(env_vars)
+        for var in unset_vars:
```

Comment:
Execution won't get here if `env_vars` is empty (if all the variables are `=None`). Please, add a test case for this.

Reply:
Good catch!!

Absolutely right. Let me fix & create a regression test case!